src/pages/integrations/shapeshift/shapeshift-confirm/shapeshift-confirm.ts

```diff
@@ -403,58 +408,56 @@ export class ShapeshiftConfirmPage {
     });
   }
 
-  private createTx(
+  private async createTx(
```

Comment:
Creation shouldn't be async. We need to have tx created before do the next steps.

Reply:
changed back to Promise based